### DISCUSSION TITLE
Download for Udemy not working

### DISCUSSION THREAD

**Discussion by Larkaji** [aclap-dev/video-downloadhelper#1012] [Help and Q&A]
### Your question

I have purchased the license to download videos from Udemy. Initially I was easily downloading each and everything from there. But now, everytime when I try to download, it didn't rahter ask for generating a error report. Is there any solution. 

### Addon & CoApp version and details

I have purchased the license to download videos from Udemy. Initially I was easily downloading each and everything from there. But now, everytime when I try to download, it didn't rahter ask for generating a error report. Is there any solution. 
<img width="254" alt="REport" src="https://github.com/user-attachments/assets/4339f2aa-14cb-43c2-8b3a-f6b8eba54d71">

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Search this forum using `udemy` as your search key.

**Reply by Larkaji** [reply]
Ok dear.

**Reply by devinschumacher** [reply]
then why does their website say they support downloading from Udemy? https://www.downloadhelper.net/asite/udemy

**Comment by paulrouget** [comment]
We found out the issue.

We are working on a fix.

**Reply by devinschumacher** [reply]
update on the fix?

**Comment by azizloj** [comment]
yeah I just noticed it stoped working on Udemy, pretty sad

**Comment by devinschumacher** [comment]
bump
